typescript/lib/services/container_run_config.ts

```diff
@@ -0,0 +1,73 @@
+// ====================================================================================================
+//                                    Config Object
+// ====================================================================================================
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+export class ContainerRunConfig {
+	private readonly entrypointOverrideArgs: string[];
+	private readonly cmdOverrideArgs: string[];
+	private readonly environmentVariableOverrides: Map<string,string>;
+	
+	constructor (
+        entrypointOverrideArgs: string[],
+        cmdOverrideArgs: string[],
+        environmentVariableOverrides: Map<string, string>) {
```

Comment:
Need some distinction between args & code (either 2-level args & 1-level code, or the `) {` separator)